source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -333,7 +335,15 @@ def _getColumnContentRaw(self, index):
 		return buffer.value if buffer else None
 
 	def _getColumnContent(self, column):
-		return self._getColumnContentRaw(self.parent._columnOrderArray[column - 1])
+		targetColumn = self.parent._columnOrderArray[column - 1]
+		try:
+			return self._getColumnContentRaw(targetColumn)
+		except ArgumentError:

```

Comment:
Could you please log a debugWarning here? Like:

```
except ArgumentError as e:
			log.debugWarning("Could not retrieve content for column %d, %r" % (column, e))
```

This might be a bit obnoxious in the log though. May be log.debug would be enough here, I'm not sure. @MichaelDCurran might have some thoughts about this.